src/Components/Onboarding/Steps/Artists/ItemLink.tsx

```diff
@@ -84,8 +78,8 @@ class ItemLink extends React.Component<Props, null> {
           unfollow: false,
         },
       },
-      updater: storeUpdater,
-      // optimisticUpdater: storeUpdater,
+      updater: (store: RecordSourceSelectorProxy, data: SelectorData) =>
+        this.props.itemLinkUpdater.onArtistFollowed(artist, store, data),
```

Comment:
…following up on the `implements` comment, this should probably just be `this.props.onArtistFollowed(artist.__id, store, data)`